### ISSUE TITLE
Faster RCNN training result prblem

### ISSUE THREAD

**Issue by hktxt** [chainer/chainercv#898]
I tried to train Faster-RCNN from scratch. The train.py gives me this result:  

  {
        "main/rpn_loc_loss": 0.045347146689891815,
        "main/rpn_cls_loss": 0.17269396781921387,
        "main/roi_loc_loss": 0.27309778332710266,
        "main/roi_cls_loss": 0.49079766869544983,
        "main/loss": 0.9819365739822388,
        "lr": 0.0001,
        "validation/main/map": 0.12205098147362133,
        "validation/main/ap/aeroplane": 0.2037050348560657,
        "validation/main/ap/bicycle": 0.15355473865782293,
        "validation/main/ap/bird": 0.1069814741360702,
        "validation/main/ap/boat": 0.08437066002057338,
        "validation/main/ap/bottle": 0.0303030303030303,
        "validation/main/ap/bus": 0.13091526733063397,
        "validation/main/ap/car": 0.2179439900145522,
        "validation/main/ap/cat": 0.1788983192876631,
        "validation/main/ap/chair": 0.027534965034965036,
        "validation/main/ap/cow": 0.10386202324629096,
        "validation/main/ap/diningtable": 0.1376211067035571,
        "validation/main/ap/dog": 0.16679011471811683,
        "validation/main/ap/horse": 0.18906283713577696,
        "validation/main/ap/motorbike": 0.22127384038737857,
        "validation/main/ap/person": 0.20588034722737675,
        "validation/main/ap/pottedplant": 0.0101010101010101,
        "validation/main/ap/sheep": 0.006993006993006993,
        "validation/main/ap/sofa": 0.03462976657260593,
        "validation/main/ap/train": 0.13475834327905686,
        "validation/main/ap/tvmonitor": 0.09583975346687211,
        "epoch": 13,
        "iteration": 70000,
        "elapsed_time": 43275.40101284254
    }

the **mAP** is extremely low... **0.12**...
I changed nothing, only the path of VOC dataset instead of download.
I'am sure the test dataset returns use_difficult=True, return_difficult=True.
and DetectionVOCEvaluator() uses use_07_metric=True.
These are default setting in train.py.
I check some issues, some had a lower mAP than reported. However, my result is wrong...
I did not use the pretrained model. To confirm evaluate code, I downloaded the pretrained model  from: 'https://chainercv-models.preferred.jp/faster_rcnn_vgg16_voc07_trained_2018_06_01.npz' and tested it, had a result of **0.706**.
So, it seems like training porblem. Is the epoch of 13 not enough? should I train more epoches?

**Comment by Hakuyume** [comment]
Could you share the version of ChainerCV and the commit id of the training code?

**Comment by hktxt** [comment]
@Hakuyume 
The version is v6.
I cloned ChainerCV repo yesterday. So it's the latest commit. 
https://github.com/chainer/chainercv/commit/c22c4512b1d9d3b8918804dafba1e8d260586321
